app/design/adminhtml/default/default/template/sales/order/create/giftmessage.phtml

```diff
@@ -13,7 +13,7 @@
  * @license     https://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
  */
 ?>
-<?php if ($this->helper('giftmessage/message')->getIsMessagesAvailable('main', $this->getQuote(), $this->getStoreId())): ?>
+<?php if ($this->canDisplayGiftmessage()): ?>
```

Comment:
```suggestion
<?php
/**
 * @var Mage_Adminhtml_Block_Sales_Order_Create_Giftmessage $this 
 */
?>
<?php if ($this->canDisplayGiftmessage()): ?>
```

Reply:
i couldn't commit the suggestion, so i did it manually